Makefile

```diff
@@ -0,0 +1,19 @@
+PROJECT  := infobloxopen/buildtool
+
+# The list of buildtool versions we are going to build,
+# make all versions as phony rules, so we will rebuild
+# them every time.
+VERSIONS := 0.0.1
```

Comment:
With the ```latest``` versioning it will be only possible to build projects that are compatible with this  one version. Say, we updated the protobuf compiler that is no longer compatible with the previous versions, with the ```latest``` tag, it would be, obviously, impossible to use this image.

I will reduce the versioning schema to ```v1```, ```v2```, but I don't see a lot of profit in this approach, instead it seems to me, that it would better keep in consistency the way we are using for versioning (semantic versioning), instead of introducing a new schema for each project.